Add guides for Cypress Accessibility (#6053)
cypress-io/cypress-documentation: docs/accessibility/guides/introduction.mdx

---
title: Introduction
sidebar_position: 10
---

:::info

##### <Icon name="question-circle" color="#4BBFD2" /> What you'll learn

- How to set goals and make progress using Cypress Accessibility
- How to generate reports during local development cycles
- Common issues and questions related to all kinds of accessibility automation
- How to use Cypress Accessibility to monitor a production website
- Switching between "improvement" and "maintenance" mindsets
  :::

## Making the most of Cypress Accessibility

> In these guides, we'll describe how to leverage Cypress Accessibility to improve and maintain the accessibility of the websites and applications you test with Cypress. We'll also discuss how to work with your Cypress point-of-contact to set the product up to provide the most value to you, per project, based on the goals you have for each specific app you are testing.

Reviewer: ```suggestion
In these guides, we'll describe how to leverage Cypress Accessibility to improve and maintain the accessibility of the websites and applications you test with Cypress. We'll also discuss how to work with your Cypress point-of-contact to set the product up to provide the most value to you based on the goals you have for each specific app you are testing.
```

Every project will be in a different state when you start to use Cypress Accessibility. You might find you have thousands of issues on hundreds of different pages and components, or you might have just a few issues detected in an already-accessible experience.

For projects that have a high volume of issues, we'll describe how to use Cypress Accessibility to drive the process of remediation and improvement, get some measurable wins on the board for your team, and support the gradual introduction of accessibility standards.

> For projects with few or no issues currently, which is the desired end state that “improving” projects are intending to get to, we will lay out how Cypress supports you to have a resilient, efficient, and timely approach to maintaining your standards and avoiding surprises over the long term.

Reviewer: ```suggestion
For projects with few or no issues currently, we’ll lay out how Cypress supports a resilient, efficient, and timely approach to maintain your standards and avoid surprises over the long term.
```
